Dockerfile

```diff
@@ -21,11 +21,15 @@ RUN yum install -y epel-release && \
     chmod -R 777 /supervisord.log /supervisord.pid /var/log/nagios \
                  /etc/httpd /etc/passwd /var/log /etc/nagios /usr/lib64/nagios \
                  /var/spool/nagios /run /usr/share/httpd /usr/share/nagios && \
-    sed -i -e 's|cfg_file=/etc/nagios/objects/localhost.cfg||' /etc/nagios/nagios.cfg
+    sed -i -e 's|cfg_file=/etc/nagios/objects/localhost.cfg||' /etc/nagios/nagios.cfg && \
+    pip install pymongo
 
 ADD supervisord.conf /etc/supervisord.conf
 ADD make-nagios-fhservices-cfg make-nagios-commands-cfg fhservices.cfg.j2 commands.cfg.j2 /opt/rhmap/
-ADD plugins/default/* plugins/core/* /opt/rhmap/nagios/plugins/
+ADD plugins/default/* \
+    plugins/core/* \
+    https://raw.githubusercontent.com/mzupan/nagios-plugin-mongodb/master/check_mongodb.py \
```

Comment:
So the BSD license which that script is released under can pretty much be included in any other license so long as the copywrite is maintained.  I'll include the file as is in our current code, but if we end up going the python route for our nagios checks, it would probably be much easier to just implement the checks we want directly instead of having a python script calling another python script via shell command.